### ISSUE TITLE
Docker Desktop: Dev Environments white blank page

### ISSUE THREAD

**Issue by larsonnn** [docker/for-mac#5808]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: F9710835-3C79-477A-BDE0-7BCD87A04271/20210630200642

### Expected behavior

Show me the Dev Environment View

### Actual behavior

blank page

### Information

- Is it reproducible? I don't know couldn't fix it yet
- Is the problem new? Yes, since I updated to latest version and created and delete one dev environment 
- Did the problem appear with an update? IDK

  - macOS Version: 11.4 (20F71)
  - Intel chip or Apple chip: Apple Chip
  - Docker Desktop Version: 3.5.1 (66090)

### Error Message in developer tools

```
main.js:2 TypeError: Cannot read property 'length' of null
    at Pj (114.bundle.js:1)
    at Vo (main.js:2)
    at ka (main.js:2)
    at vc (main.js:2)
    at Ds (main.js:2)
    at Cs (main.js:2)
    at rs (main.js:2)
    at main.js:2
    at e.unstable_runWithPriority (main.js:2)
    at Gn (main.js:2)
```
![Bildschirmfoto 2021-06-30 um 22 11 26](https://user-images.githubusercontent.com/18642095/124024924-215c2f00-d9f0-11eb-9892-6e62a107bbb6.png)

**Comment by gtardif** [comment]
Thx @larsonnn for the feedback and detailed info. 

This is a duplicate of https://github.com/docker/dev-environments/issues/4, and will be fixed in the next release. 
In the meantime, you can unblock yourself by removing the Dev Environment file that causes the issue : `rm ~/.docker/devenvironments/data.json` should allow you to go back to the Dev Environment screen. 

(sorry you will loose previous Dev Environment you might have created ; please do not hit the "remove" button while a Dev Environment is being created, this would lead you to the blank screen again with your current version. )

Closing this one as a duplicate.
